sdk/search/azure-search-documents/dev_requirements.txt

```diff
@@ -1,5 +1,6 @@
 -e ../../../tools/azure-devtools
 -e ../../../tools/azure-sdk-tools
 ../../core/azure-core
+../azure-search-nspkg
```

Comment:
Should this be azure-search-documents-nspkg?

Reply:
Not AFAIK, this is the package that sets up `azure.search` as a place that other packages (e.g. azure-search-documents) can separately install into.

cc @johanste

Reply:
Dev files should not need nspkg

Reply:
FWIW I was following this example

https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/keyvault/azure-keyvault-secrets/dev_requirements.txt#L6